spec.md

```diff
@@ -592,6 +628,50 @@ message CreateVolumeRequest {
   // logged, etc.) by the CO.
   // This field is OPTIONAL.
   map<string, string> controller_create_secrets = 5;
+
+  // Contains all attributes of the source that is used to create
+  // the volume from. This field is OPTIONAL.
+  ContentSource content_source = 6;
+}
+
+// Specifies what source the volume will be created from. One of the
+// source_type fields MUST be specified. Only SnapshotReference is
+// a supported ContentSource at the moment.
+message ContentSource {
+  oneof source_type {
+    SnapshotReference snapshot_ref = 1;
```

Comment:
Sure.

Reply:
Since I renamed SnapshotReference to Snapshot and defined it as a nested type inside VolumeContentSource, I can just call it snapshot instead of snapshot_reference?